### PR TITLE
Expose exit codes

### PR DESCRIPTION
**Pull request by munkyboy**

### CODE REVIEW

File: blackbox-test/features/dc.feature [hunk 1]
```diff
@@ -14,12 +14,22 @@
...
-  Scenario: with a standard project
-    Given a file named "docker-compose.yml" with:
-    """yaml
-    foo:
-      image: busybox
-      command: echo bar
-    """
-    When I run `lc dc -- run --rm foo`
-    Then it should succeed with "bar"
+Scenario: with a standard project
```
Comment (paulcichonski) [inline blackbox-test/features/dc.feature]: nit: indent change seems un-needed.

Comment (munkyboy) [reply blackbox-test/features/dc.feature]: nit: I hate your indents

File: vendor/vendor.json [hunk 2]
```diff
@@ -21,10 +21,10 @@
 			"revisionTime": "2017-04-08T17:35:22Z"
 		},
 		{
-			"checksumSHA1": "i27YHcyHY4stkXm12/ztso3wHUo=",
+			"checksumSHA1": "h4FKEw19jw7+Je5OwsISRKIGTm8=",
 			"path": "github.com/codegangsta/cli",
```
Comment (paulcichonski) [inline vendor/vendor.json]: it seems like this tool had a name change, this now re-directs to `https://github.com/urfave/cli`, should we update it here?

Comment (munkyboy) [reply vendor/vendor.json]: I was able to fetch it fine. Not sure how changing the name here would affect go imports

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## v4.0.0
```
Comment (munkyboy) [inline CHANGELOG.md]: I doubt this would break any consumers, but I thought I would play it safe.

File: helpers/command.go [hunk 4]
```diff
@@ -28,10 +30,27 @@ func RunCommand(command *exec.Cmd) error {
...
 	logrus.Debugf("running command %s with args %v", command.Path, command.Args)
-	if err := command.Run(); err != nil {
-		logrus.Debug("last command was not successful")
+
+	if err := command.Start(); err != nil {
+		logrus.Error("unable to start command")
 		return err
 	}
+
+	if err := command.Wait(); err != nil {
```
Comment (paulcichonski) [inline helpers/command.go]: `command.Run` also returns `*ExitError`: https://golang.org/pkg/os/exec/#Cmd.Run, why not just use that?

Comment (munkyboy) [reply helpers/command.go]: so we don't try to extract exit code in the case that `Start()` returned an error

File: CHANGELOG.md (hunk 3 above)
Comment (paulcichonski) [reply CHANGELOG.md]: reminder: when doing the releasing you have to create the tag directly on this `cisco/elsy`, and then don't touch the temp github release object. Travis will eventually overwrite it with the binaries when it finishes building the tag.

**Pull request merged**